libs/barista-components/drawer-table/README.md

```diff
@@ -0,0 +1,71 @@
+# Drawer table
+
+<ba-live-example name="DtExampleDrawerTableDefault" fullwidth></ba-live-example>
+
+The `DtDrawerTable` component uses table and drawer components. Table is the
+main content of the drawer and row click of the table opens the drawer content
+of the row.
+
+## Imports
+
+You have to import the `DtDrawerTableModule` to use the `dt-drawer-table`.
+
+```typescript
+import { NgModule } from '@angular/core';
+import { DtDrawerTableModule } from '@dynatrace/barista-components/drawer-table';
+
+@NgModule({
+  imports: [DtTableModule],
+})
+class MyModule {}
+```
+
+## Inputs
+
+The `DtDrawerTable` component supports the following input.
+
+| Name          | Type       | Default      | Description                                                    |
+| ------------- | ---------- | ------------ | -------------------------------------------------------------- |
+| `openColumns` | `string[]` | first column | Defines the columns that stay visible when the drawer is open. |
+
+#### Methods
+
+The following methods are on the `DtDrawerTable` class:
+
+| Name     | Description        | Return value |
+| -------- | ------------------ | ------------ |
+| `open`   | Opens the drawer   | `void`       |
+| `close`  | Closes the drawer  | `void`       |
+| `toggle` | Toggles the drawer | `void`       |
+
+## Basic example
+
+**An ng-template that defines the drawer's content has to be the first child of
+the component**
```

Comment:
it doesn't necessarily need to be the first child of the component. It's defined as a content child for the drawer-table, which requires it to be a direct child of the drawer table, but the order should be irrelevant.

Reply:
Yes it is strange, but it doesn't work when it isn't the first child. I have checked once more now to be sure and it couldn't find the template when it is defined after the table.

Reply:
This is definitely weird. This should work and probably requires some investigation. I guess you have found some weird behavior on this one. I will have a look on monday to see, why it will not find the template if it's not the first child.

Reply:
If you declare your own directive for the templateRef it should be fine, wherever it is. See https://github.com/dynatrace-oss/barista/pull/663/files#r392887831

The ContentChild will look for the first templateRef match, and since some directives from the table do extend from it, it is quite possible to get a wrong match. 
to be safe here, you can define your own directive and read the templateRef.

Reply:
If you declare your own directive for the templateRef it should be fine, wherever it is. See https://github.com/dynatrace-oss/barista/pull/663/files#r392887831

The ContentChild will look for the first templateRef match, and since some directives from the table do extend from it, it is quite possible to get a wrong match. 
to be safe here, you can define your own directive and read the templateRef.